Shared/Models/NftMetadata.cs

```diff
@@ -33,5 +33,13 @@ public class NftMetadata
         }
 
         public string? contentType { get; set; }
+
+        public List<Trait>? attributes { get; set; }
```

Comment:
if the field is called attributes, why call the elements Trait? Why not something like NftAttribute?

Reply:
ive renamed the trait class to nftattribute so it makes sense